packages/cli-kit/src/git.ts

```diff
@@ -12,12 +13,61 @@ export const GitNotPresentError = () => {
   )
 }
 
+export const OutsideGitDirectoryError = () => {
```

Comment:
I'd include the directory in which we are trying to run a Git command. That'll make it easier to debug issues that arise.